internal/hammer/diff.go

```diff
@@ -40,6 +40,17 @@ func NewDatabase(ddl DDL) (*Database, error) {
 			} else {
 				return nil, fmt.Errorf("cannot find ddl of table to apply index %s", stmt.Name)
 			}
+		case *spansql.AlterTable:
```

Comment:
I think adding a foreign key should be expressed in a create table, just like adding a column. So I don't think we need to support alter tables, what do you think?

Reply:
Spanner's [DDL allows it](https://cloud.google.com/spanner/docs/data-definition-language#alter_table), but also retrieving a ddl from spanner relies on this feature unfortunately which makes supporting it a requirement. For example running `gcloud spanner databases ddl  describe db --instance=test` on a db with a table that has an FK, spanner returns two statements 1 `CREATE TABLE` and a following `ALTER TABLE` to add the FK constraint.

Reply:
I understand. Thank you very much for checking the detailed cases !